OpenMEEG/libs/OpenMEEG/include/operators.h

```diff
@@ -180,7 +180,10 @@ namespace OpenMEEG {
     }
 
     inline double _operatorP1P0(const Triangle& T2,const Vertex& V1) {
-        return (T2.contains(V1)) ? 0.0 : T2.area()/3.0;
+        double result = 0.;
+        if  (T2.contains(V1))
+            result = T2.area()/3.0;
+        return result;
     }
```

Comment:
of for you or shall I change that into:
return (T2.contains(V1))?T2.area()/3.0:0.;
as Theo suggest ?